cmd/manager/main.go

```diff
@@ -110,6 +111,25 @@ func main() {
 		os.Exit(1)
 	}
 
+	//Setup TrainedModel controller
+	trainedModelEventBroadcaster := record.NewBroadcaster()
+	setupLog.Info("Setting up v1beta1 TrainedModel controller")
+	if err != nil {
```

Comment:
You've already checked the error here on line 109

Reply:
Good catch!